fili-core/src/main/java/com/yahoo/bard/webservice/util/CompletedFuture.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.util;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * A future response which has already been completed and will return
+ * as soon as it is called.
+ *
+ * @param <E>  The type to be returned by this future
+ */
+public class CompletedFuture<E> implements Future<E> {
+    private final E item;
+    private final Throwable throwable;
+
+    /**
+     * Creates a completed future with an item to return or
```

Comment:
I think this is clear from the context and adding this could make it unclear what happens when `{@link #get(long, TimeUnit)}` is called.